pallets/core/src/modules/trust_registry/mod.rs

```diff
@@ -288,9 +310,13 @@ pub mod pallet {
         ) -> DispatchResult {
             ensure_signed(origin)?;
 
+            let f = ActionWithNonceWrapper::wrap_fn_with_modify_removable_action(
```

Comment:
Can this lead to removing the registry? If not, how do we remove?

Reply:
For now, the convener can't obliterate the registry but he can remove all schemas metadata.